core/data-resolver/types/DataResolver.d.ts

```diff
@@ -8,7 +8,7 @@ import Review from 'core/modules/review/types/Review';
 declare namespace DataResolver {
 
   interface CategorySearchOptions {
-    parentId?: number,
+    parentId?: any,
```

Comment:
In this case, these types should be number|string rather than any.

Reply:
Makes sense @lukeromanowicz. These changes can be applied to all the interface files. WDYT?

Reply:
I have updated the PR. Please check.